indico/modules/rb/__init__.py

```diff
@@ -70,3 +72,42 @@ def _event_deleted(event, user, **kwargs):
     for resv in reservations:
         resv.event_id = None
         resv.cancel(user or session.user, 'Associated event was deleted')
+
+
+@signals.menu.sections.connect_via('rb-sidemenu')
+def _sidemenu_sections(sender, **kwargs):
+    yield 'search', SideMenuSection(_("Search"), 40, icon='search', active=True)
+    yield 'my_rooms', SideMenuSection(_("My Rooms"), 30, icon='user')
+    yield 'blocking', SideMenuSection(_("Room Blocking"), 20, icon='lock')
+
+
+@signals.menu.items.connect_via('rb-sidemenu')
+def _sidemenu_items(sender, **kwargs):
+    user_is_admin = session.user is not None and rb_is_admin(session.user)
+    map_available = Location.default_location is not None and Location.default_location.is_map_available
+
+    yield 'book_room', SideMenuItem(_('Book a Room'), url_for('rooms.book'), 80, icon='checkmark')
+    yield 'map', SideMenuItem(_('Map of Rooms'), url_for('rooms.roomBooking-mapOfRooms'), 70, icon='location',
+                              visible=map_available)
+    yield 'calendar', SideMenuItem(_('Calendar'), url_for('rooms.calendar'), 60, icon='calendar')
+    yield 'my_bookings', SideMenuItem(_('My Bookings'), url_for('rooms.my_bookings'), 50, icon='time')
+    yield 'search_bookings', SideMenuItem(_('Search bookings'), url_for('rooms.roomBooking-search4Bookings'),
+                                          section='search')
+    yield 'search_rooms', SideMenuItem(_('Search rooms'), url_for('rooms.search_rooms'),
+                                       section='search')
+    yield 'book_in_my_rooms', SideMenuItem(_('Bookings in my rooms'), url_for('rooms.bookings_my_rooms'),
```

Comment:
`bookings_` would fit better - `book_` sounds like it's a page to book stuff

Also, those items used to be disabled when you don't own any rooms. Maybe we could hide the whole "my rooms" section if you don't manage any rooms?